redisdb/manifest.json

```diff
@@ -11,5 +11,6 @@
     "windows"
   ],
   "version": "0.1.0",
-  "guid": "0e2f3ed1-d36b-47a4-b69c-fedb50adf240"
+  "guid": "0e2f3ed1-d36b-47a4-b69c-fedb50adf240",
+  "max_agent_version": "6.0.0"
```

Comment:
Could we maybe assume on the omnibus side that if no `max_agent_version` is specified that it's not upper bound by version?